Tests/Realm.Tests/Database/LifetimeTests.cs

```diff
@@ -34,8 +35,6 @@ private static WeakReference GetWeakRealm()
         [Test]
         public void RealmObjectsShouldKeepRealmAlive()
         {
-            TestHelpers.IgnoreOnWindows("GC blocks on Windows");
```

Comment:
How were this test and a few others below fixed when in the test nothing is changed? What am I missing?

Reply:
Likely fixed when we pulled in the new conditional variable implementation. There's nothing in this PR that fixes them - just noticed they're passing while reenabling all tests in Windows.